cmd/kops/create_cluster.go

```diff
@@ -59,6 +59,8 @@ type CreateClusterOptions struct {
 	MasterSize           string
 	MasterCount          int32
 	NodeCount            int32
+	MasterVolumeSize     int32
```

Comment:
@matthew-marchetti as I mentioned above, if we do not parse MB or TB we need to mention what the number size is based upon in the Cobra CLI help.

>I understand, we need to document size of the int then. Meaning are we defining mb or gb in a int. We can do a follow-up pr.

Please figure out what measurement AWS is defaulting to and document.  Get that done and it looks good.  I think there is an issue open for increasing the default volume size if that was not done already.

Thanks!